README.md

```diff
@@ -129,19 +129,19 @@ Let's see how **ExShards** works in distributed fashion.
 Node `a`:
 
 ```
-$ iex --sname a@localhost -S mix
+$ iex --name a@localhost -S mix
 ```
 
 Node `b`:
 
 ```
-$ iex --sname b@localhost -S mix
+$ iex --name b@localhost -S mix
 ```
 
 Node `c`:
 
 ```
-$ iex --sname c@localhost -S mix
+$ iex --name c@localhost -S mix
```

Comment:
Besides, in the next step (step 2) you should also change this:

`ExShards.new :mytab, scope: :g, nodes: [:b@localhost, :c@localhost]`

By

`ExShards.new :mytab, scope: :g, nodes: [:"b@127.0.0.1", :"c@127.0.0.1"]`